packages/atomic/src/components/search/tabs/atomic-tab-manager/e2e/atomic-tab-manager.e2e.ts

```diff
@@ -44,7 +44,7 @@ test.describe('AtomicTabManager', () => {
       test.beforeEach(async ({facets}) => {
         await facets.getFacetValue.first().waitFor({state: 'visible'});
       });
-      test('facets', async ({tabManager}) => {
+      test.fixme('facets', async ({tabManager}) => {
```

Comment:
Why ?

Reply:
Something weird happened with the rebase, I'm seeing a bunch of unrelated files included, will fix that